### PR TITLE
Config file

### PR DESCRIPTION
**Pull request by daktak**
config file supports
```
input:
camera:
model:
output:
name:
date:
buffer:
prefix:
range:
skip_aux:
sort_by:
connection:
```

### CODE REVIEW

File: cmd/import.go [hunk 1]
```diff
@@ -22,44 +22,19 @@ var importCmd = &cobra.Command{
...
-			cui.Error("Problem parsing camera", err)
-		}
-		projectName, err := cmd.Flags().GetString("name")
-		if err != nil {
-			cui.Error("Problem parsing name", err)
-		}
+                input := get_flag_string(cmd, "input")
+                output := get_flag_string(cmd, "output")
+                camera := get_flag_string(cmd, "camera")
+                projectName := get_flag_string(cmd, "name")
```
Comment (KonradIT) [inline cmd/import.go:28]: What do you think about making `--name` mandatory to be passed via CLI each time? So videos don't get accidentally sent to an old project, for instance.

**Review by KonradIT — APPROVED** [review]

File: cmd/import.go (hunk 1 above)
Comment (daktak) [reply cmd/import.go:28]: If a user doesn't have "projects?" and uses camera/date or even the video file name?
maybe it should be combination with something?

**Comment by KonradIT** [comment]
Looks good, merging but will add a few additions on top.

File: cmd/import.go (hunk 1 above)
Comment (KonradIT) [reply cmd/import.go:28]: True, lets keep it the way it is.

**Pull request merged**